packages/device_info_plus/device_info_plus/android/src/main/java/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.java

```diff
@@ -98,10 +98,8 @@ private String[] getSystemFeatures() {
   }
 
   /**
-   * Returns the Android hardware device ID that is unique between the device + user and app
-   * signing. This key will change if the app is uninstalled or its data is cleared. Device factory
-   * reset will also result in a value change.
-   *
+   * Returns the Android hardware device ID that is unique between the device + user and app signing.
+   * This key will change if a factory reset is performed on the device or if an APK signing key changes.
```

Comment:
I think it is still not fully correct and we should also mention the difference for devices with Android versions before 8 and those with 8 and higher.

Suggest something like:
```
On devices with Android >= 8.0 (API level 26) returns an ID that is unique to each combination of app-signing key, user, and device. The value may change if a factory reset is performed on the device or if an APK signing key changes.
On devices with Android < 8.0 (API level 26) returns an ID that is randomly generated when the user first sets up the device and should remain constant for the lifetime of the user's device.
```

Reply:
@martinoyovo You might have missed this comment ☝🏻